src/compiler/checker.ts

```diff
@@ -4167,12 +4177,12 @@ module ts {
                 var propType = getTypeOfSymbol(p);
                 var widenedType = getWidenedType(propType);
                 if (propType !== widenedType) {
-                    var symbol = <TransientSymbol>createSymbol(p.flags | SymbolFlags.Transient, p.name);
-                    symbol.declarations = p.declarations;
-                    symbol.parent = p.parent;
+                var symbol = <TransientSymbol>createSymbol(p.flags | SymbolFlags.Transient, p.name);
+                symbol.declarations = p.declarations;
+                symbol.parent = p.parent;
                     symbol.type = widenedType;
-                    symbol.target = p;
-                    if (p.valueDeclaration) symbol.valueDeclaration = p.valueDeclaration;
+                symbol.target = p;
+                if (p.valueDeclaration) symbol.valueDeclaration = p.valueDeclaration;

```

Comment:
This is an issue with the VS diff/merge tool. It seemed to be ignoring whitespace differences during any automatic merge operations.